channels/applicationinsights-channel-js/src/Statsbeat.ts

```diff
@@ -0,0 +1,203 @@
+import { NetworkStatsbeat } from "./NetworkStatsbeat";
+import { Sender } from "./Sender";
+import { StatsbeatCounter } from "./Constants";
+import { IConfig, Metric } from "@microsoft/applicationinsights-common";
+import { IConfiguration, IAppInsightsCore, IPlugin, ITelemetryPluginChain, ITelemetryItem, objKeys, dateNow,  } from '@microsoft/applicationinsights-core-js';
+import dynamicProto from '@microsoft/dynamicproto-js';
+
+export class Statsbeat {
+    public static INSTRUMENTATION_KEY = "c4a29126-a7cb-47e5-b348-11414998b11e";
```

Comment:
Any reason for these to be public?

Reply:
Nice catch, we should just make this a const at the top of the file.

Reply:
Thanks, modified.